File: Python/1037.py
# -*- coding: utf-8 -*-
n = float(input())
        
if n >= 0 and n <= 25:
    print("Intervalo [0,25]")
        
elif n > 25 and n <= 50:
    print("Intervalo (25,50]")
        
elif n > 50 and n <= 75:
    print("Intervalo (50,75]")
        
elif n > 75 and n <= 100:
    print("Intervalo (75,100]")
        
else:
    print("Fora de intervalo")

File: Java/1037.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        
        double n = leitor.nextDouble();
        
        if (n >= 0 && n <= 25) {
            System.out.println("Intervalo [0,25]");
        }
        else if (n > 25 && n <= 50) {
            System.out.println("Intervalo (25,50]");
        }
        else if (n > 50 && n <= 75) {
            System.out.println("Intervalo (50,75]");
        }
        else if (n > 75 && n <= 100) {
            System.out.println("Intervalo (75,100]");
        }
        else {
            System.out.println("Fora de intervalo");
        }
    }
 
}